sympy/combinatorics/coset_table.py

```diff
@@ -813,14 +824,27 @@ def conjugates(self, R):
 
     def modified_define(self, alpha, x):
         '''
-        Input:  ∈ , x∈A, with  x undefined.
+        Input: \alpha \epsilon \Omega, x \epsilon A*
+
+        Summary
+        =======
+        Define a function p_p from from [1..n] to A* as
+        an additional component of the modifief coset table.
```

Comment:
typo: "modified". Also, I think this line starting with "Define" should be the first sentence of the docstring.